label_sync/README.md

```diff
@@ -0,0 +1,16 @@
+# label_sync
+
+kubeflow_label.yml: containing rules to update or migrate github labels on repos in kubeflow org
+We use same tool from kubernetes: https://github.com/kubernetes/test-infra/tree/master/label_sync
+
+Want to edit kubeflow_label.yml? Search process\_label.py (hacky) in this repo.
```

Comment:
You can just link to `process_label.py` using the syntax 

`[process_label.py](../hack/label_generate/process_label.py)`